manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         opendatahub.io/notebook-python-dependencies: '[{"name":"code-sever","version":"4.16"}]'
         openshift.io/imported-from: quay.io/opendatahub/workbench-images
         opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/notebook-build-commit: "1c5bcbfc222d"
```

Comment:
Uf, this is very error prone - I think that we can utilize the relevant `odh-minimal-notebook-image-n` variables that are stored in `params.env` file. Ideally cut the hashes to 12 digits as used here.